c2corg_api/emails/email_service.py

```diff
@@ -20,8 +23,9 @@ def _get_file_content(self, lang, key):
         filepath = os.path.dirname(__file__) + '/i18n/%s/%s' % (lang, key)
         if not os.path.isfile(filepath):
             filepath = os.path.dirname(__file__) + '/i18n/%s/%s' % ('fr', key)
-        f = open(filepath, 'r')
-        return f.read().rstrip()  # No trailing new line in email subject!
+        with open(filepath, 'rU') as f:  # Use universal newline support.
+            content = f.read().rstrip()  # No trailing new line in subject!
+        return content
```

Comment:
Don't you need to move the ``return`` statement to the ``with``block where ``content`` is defined?

Reply:
Yes it should be better for the case the file does not exist.